README.md

```diff
@@ -1777,19 +1784,68 @@ NOTE: These options may be changed/removed in the future without concern for bac
+Note that **all** plugins are imported even if not invoked, and that **there are no checks** performed on plugin code. **Use plugins at your own risk and only if you trust the code!**
 
-Plugins can be of `<type>`s `extractor` or `postprocessor`. Extractor plugins do not need to be enabled from the CLI and are automatically invoked when the input URL is suitable for it. Postprocessor plugins can be invoked using `--use-postprocessor NAME`.
+Plugins can be of `<type>`s `extractor` or `postprocessor`. 
+- Extractor plugins do not need to be enabled from the CLI and are automatically invoked when the input URL is suitable for it. 
+- Extractor plugins take priority over builtin extractors.
+- Postprocessor plugins can be invoked using `--use-postprocessor NAME`.
 
-See [ytdlp_plugins](ytdlp_plugins) for example plugins.
 
-Note that **all** plugins are imported even if not invoked, and that **there are no checks** performed on plugin code. Use plugins at your own risk and only if you trust the code
+Plugins are loaded from namespace-package `ytdlp_plugins.extractor` and `ytdlp_plugins.postprocessor`.
 
-If you are a plugin author, add [ytdlp-plugins](https://github.com/topics/ytdlp-plugins) as a topic to your repository for discoverability
+In other words, the file structure on the disk looks something like:
+    
+        ytdlp_plugins/
+            extractor/
+                myplugin.py
+            postprocessor/
+                myplugin.py
+
+yt-dlp looks for these `ytdlp_plugins` folders in various locations and reads in plugins from all of them.
 
 See the [wiki for some known plugins](https://github.com/yt-dlp/yt-dlp/wiki/Plugins)
 
+## Installing Plugins
+
+Plugins can be installed in a variety of different ways. 
+
+1. **User Plugins**:
+    * `${XDG_CONFIG_HOME}/yt-dlp/plugins/<package name>/ytdlp_plugins/` (recommended on Linux/macOS)
+    * `${APPDATA}/yt-dlp/plugins/<package name>/ytdlp_plugins/` (recommended on Windows)
+    * `~/.yt-dlp/plugins/<package name>/ytdlp_plugins/`
+
+2. **System Plugins**:
+    * `/etc/yt-dlp/plugins/<package name>/ytdlp_plugins/`
+
+3. **yt-dlp executable location**:
+    * Binary: Plugins can be installed root directory of the binary (where `<root-dir>/yt-dlp.exe`, `<root-dir>/ytdlp_plugins/`)
+    * Source: plugins can be installed in the root directory of the module (where `<root dir>/yt_dlp/__main__.py`, `<root-dir>/ytdlp_plugins/`)
+
+4. **pip and other locations in `PYTHONPATH`**
+    * Plugins can be installed and managed using `pip`. See [ytdlp-sample-plugins](https://github.com/coletdjnz/ytdlp-sample-plugins) for an example.
+    * Any path in `PYTHONPATH` is searched in for `ytdlp_plugins`
+      * Note: This does not apply for Pyinstaller/py2exe builds.
```

Comment:
You can reference #configuration. Something like `Plugins can be installed into a "plugins" folder under any [configuration directory](#configutation). E.g. ...`

Reply:
Hmm possibly. I do think these should go under a configuration directory heading. 

But I also think it is good to explicit about the locations, even if it is somewhat duplicated, just to make it easier and more clear for the average user.

Could do something like:
```diff
diff --git a/README.md b/README.md
index 2204bf546..d3b01f73f 100644
--- a/README.md
+++ b/README.md
@@ -1810,14 +1810,15 @@ ## Installing Plugins
 
 Plugins can be installed in a variety of different ways. 
 
-1. **User Plugins**:
-    * `${XDG_CONFIG_HOME}/yt-dlp/plugins/<package name>/ytdlp_plugins/` (recommended on Linux/macOS)
-    * `${APPDATA}/yt-dlp/plugins/<package name>/ytdlp_plugins/` (recommended on Windows)
-    * `~/.yt-dlp/plugins/<package name>/ytdlp_plugins/`
-
-2. **System Plugins**:
-    * `/etc/yt-dlp/plugins/<package name>/ytdlp_plugins/`
-
+1. **Configuration directories**:
+    * Plugins can be installed into a `plugins/<package name>` directory under any [configuration directory](#configuration).
+        * where `<package name>` is a folder containing `ytdlp_plugins`
+    * **User Plugins**
+      * `${XDG_CONFIG_HOME}/yt-dlp/plugins/<package name>/ytdlp_plugins/` (recommended on Linux/macOS)
+      * `${APPDATA}/yt-dlp/plugins/<package name>/ytdlp_plugins/` (recommended on Windows)
+      * `~/.yt-dlp/plugins/<package name>/ytdlp_plugins/`
+    * **System Plugins**
+      * `/etc/yt-dlp/plugins/<package name>/ytdlp_plugins/`
 3. **yt-dlp executable location**:
     * Binary: Plugins can be installed root directory of the binary (where `<root-dir>/yt-dlp.exe`, `<root-dir>/ytdlp_plugins/`)
     * Source: plugins can be installed in the root directory of the module (where `<root dir>/yt_dlp/__main__.py`, `<root-dir>/ytdlp_plugins/`)
@@ -1827,7 +1828,7 @@ ## Installing Plugins
     * Any path in `PYTHONPATH` is searched in for `ytdlp_plugins`
       * Note: This does not apply for Pyinstaller/py2exe builds.
 
-(where `<package name>` is a folder containing `ytdlp_plugins`)
+
 
 .zip, .egg and .whl archives containing `ytdlp_plugins` in their root are also supported. These can be placed in the same locations `ytdlp_plugins` can be found.
 - e.g. `${XDG_CONFIG_HOME}/yt-dlp/plugins/myplugin.zip` where `myplugin.zip` contains `ytdlp_plugins/<type>/myplugin.py`
 ```